ci/pytest_errors.sh

```diff
@@ -1,15 +1,19 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+set -euo pipefail
+
 # parse the pytest log and print the pytest errors in a concise way
 
 if [[ -f "pytest.log" ]]; then
```

Comment:
Could we use a pytest plugin instead of a wrapper script so it would be available for all contributors regardless of the platform?

Reply:
The pytest result log plugin may help here but it requires a bit of munging with jq or something similar. Would it help to state the problem that the log munging is solving?

Reply:
This is now capture using a github action that will report errors in the GitHub actions UI.